libs/net/dal/Migrations/1.0.61/Up/PostUp/00-Notification.sql

```diff
@@ -0,0 +1,75 @@
+DO $$
+BEGIN
+
+UPDATE public."notification" SET
+  "settings" = regexp_replace(regexp_replace('{ "subject": "@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Notification.Models.TemplateModel>
+@using System.Linq
+@{
+  var isAV = @Content.ContentType == TNO.Entities.ContentType.AudioVideo;
+  var isTranscriptAvailable = isAV && !string.IsNullOrWhiteSpace(@Content.Body) && @Content.IsApproved;
+  var sourceCode = !string.IsNullOrEmpty(@Content.Source?.Code) ? @Content.Source.Code : @Content.OtherSource;
+  var body = isTranscriptAvailable ? @Content.Body : (isAV ? @Content.Summary : @Content.Body);
+  var transcriptIcon = isTranscriptAvailable ? \"▶️📄\" : (isAV ? \"▶️\" : \"\");
+  var toneIcon = \"\";
+  switch (@Content.TonePools.FirstOrDefault()?.Value)
+  {
+    case 0:
+      toneIcon = \"😐 \";
+      break;
+    case -3:
+    case -4:
+    case -5:
+      toneIcon = \"☹️ \";
+      break;
+    case 3:
+    case 4:
+    case 5:
+      toneIcon = \"🙂 \";
+      break;
+  }
+}@toneIcon@sourceCode: @Content.Headline @transcriptIcon" }', '[\r\t]+', '', 'g'), '[\n]+', '\\n', 'g')::jsonb
+  , "template" = '@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Notification.Models.TemplateModel>
+@{
+  var mmiaUrl = @MmiaUrl?.AbsoluteUri;
```

Comment:
Do you need to do this `AbsoluteUri`?

Reply:
Not sure if it's clear from the sql but I changed the template so that you get this now, instead of just a link to MMIA
![image](https://github.com/bcgov/tno/assets/63015960/c23ac51f-64dd-4dbc-8276-066c69903061)
The first link is mmiaUrl and the second is the deep link to the article as constructed in the `NotificationManager`